.github/workflows/build.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       GUSTO_PARALLEL_LOG: CONSOLE
       PYOP2_CFLAGS: -O0
       # Make sure that tests with >4 processes are not silently skipped
-      PYTEST_MPI_MAX_NPROCS: 4
+      PYTEST_MPI_MAX_NPROCS: 6
```

Comment:
This isn't sufficient. You also need to add extra `      - name: Run tests (nprocs = ???)` steps below.